pkg/reconciler/testing/table.go

```diff
@@ -279,17 +280,13 @@ func (r *TableRow) Test(t *testing.T, factory Factory) {
 	}
 }
 
-func filterUpdatesWithSubresource(
-	subresource string,
-	actions []clientgotesting.UpdateAction,
-) (result []clientgotesting.UpdateAction) {
-
+func filterUpdatesWithSubresource(subresource string,
+	actions []clientgotesting.UpdateAction) (result []clientgotesting.UpdateAction) {
```

Comment:
I don't like this reformatting - it's less legible

Reply:
OK, I'll do middle path. Hanging comma here looks ugly :/

Reply:
> Hanging comma here looks ugly :/ 

heh - go has many hanging commas